packages/devtools/src/transactions/format.ts

```diff
@@ -3,9 +3,11 @@ import { formatEid } from '@/omnigraph/format'
 
 export const formatOmniTransaction = (
     transaction: OmniTransaction
-): Record<string, string | number | bigint | undefined> => ({
+): Record<string, string | number | bigint | undefined | null> => ({
     Endpoint: formatEid(transaction.point.eid),
     OmniAddress: transaction.point.address,
+    OmniContract: transaction.point.contractName,
+    'Function Name': transaction.functionName,
```

Comment:
These are currently always printing empty, i will look into how to set these properties on the relevant transactions